Description: backup-manager's version string includes "-debian"
 We want people to know that it differs from upstream.
Author: Alexis Sukrieh <sukria@sukria.net>
Forwarded: not-needed

---
 backup-manager |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/backup-manager
+++ b/backup-manager
@@ -27,7 +27,7 @@
 
 RELEASE="true"
 REVISION='$Revision$'
-VERSION="0.7.11"
+VERSION="0.7.11-debian1"
 
 # All the paths we provide
 libdir="/usr/share/backup-manager"
